expenses/controllers/RequisitionController.py

```diff
@@ -1,6 +1,17 @@
 from django.db.models import Max
 from itertools import zip_longest
-from expenses.models import Requisition, Project, Vendor, RequisitionItem
+from expenses.models import Requisition, Project, Vendor, RequisitionItem, File
+from expenses.config import bucket
+import time
+
+
+def upload_file(file, requisition):
+    name = file["name"]
```

Comment:
You should check the file type here as well as a general precaution

Reply:
Fixed

Reply:
Do you validate file size anywhere here?